### PR TITLE
Introduce points input functionality on view

### PR DESCRIPTION
**Pull request by IvanYarochkin**

### CODE REVIEW

File: src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+    private String timeConcentrationPairsJson;
     private double riverSpeed;
-    private double radius;
-    private double time;
     private double lat;
     private double lng;
 
-    public double getWindSpeed() {
-        return windSpeed;
+    public InputData() {
```
Comment (xxMikhailx) [inline src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java:13]: This logic must be in enpoint of controller for GET this page.

File: src/main/resources/templates/fragment/head.html [hunk 2]
```diff
@@ -14,6 +14,7 @@
     <link th:href="@{/webjars/font-awesome/5.8.1/css/all.css}" type="text/css" rel="stylesheet" media="screen,projection"/>
     <link th:href="@{/webjars/leaflet/1.4.0/leaflet.css}" type="text/css" rel="stylesheet" media="screen,projection"/>
     <link th:href="@{/css/style.css}" type="text/css" rel="stylesheet" media="screen,projection"/>
+    <link th:href="@{https://use.fontawesome.com/releases/v5.8.2/css/all.css}" rel="stylesheet" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
```
Comment (xxMikhailx) [inline src/main/resources/templates/fragment/head.html]: Delete redundant library. Duplicate

File: src/main/resources/templates/fragment/script.html [hunk 3]
```diff
@@ -10,6 +10,8 @@
     <script th:src="@{/webjars/bootstrap/4.3.1/js/bootstrap.min.js}"></script>
     <script th:src="@{/webjars/leaflet/1.4.0/leaflet.js}"></script>
     <script th:src="@{/js/turf.min.js}"></script>
+    <script th:src="@{/js/map.utils.js}"></script>
+    <script th:src="@{/js/validation.js}"></script>
```
Comment (xxMikhailx) [inline src/main/resources/templates/fragment/script.html]: Are you sure this 2 scripts needed for all pages or just for index?

**Review by xxMikhailx — CHANGES_REQUESTED** [review]

File: src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java (hunk 1 above)
Comment (IvanYarochkin) [reply src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java:13]: Done.

File: src/main/resources/templates/fragment/head.html (hunk 2 above)
Comment (IvanYarochkin) [reply src/main/resources/templates/fragment/head.html]: Done.

File: src/main/resources/templates/fragment/script.html (hunk 3 above)
Comment (IvanYarochkin) [reply src/main/resources/templates/fragment/script.html]: Done.

File: src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java (hunk 1 above)
Comment (xxMikhailx) [reply src/main/java/by/litelife/mk/pollutanttransport/model/InputData.java:13]: Delete empty constructor since you have default.

**Review by xxMikhailx — APPROVED** [review]

**Pull request merged**